package.json

```diff
@@ -13,22 +13,24 @@
         "start:server": "tinylicious"
     },
     "dependencies": {
-        "@fluid-experimental/data-objects": "^0.34.0-14942",
-        "@fluidframework/aqueduct": "^0.32.1",
-        "@fluidframework/get-tinylicious-container": "^0.32.1",
-        "@fluidframework/map": "^0.32.1",
+        "@fluid-experimental/data-objects": "^0.35.2",
+        "@fluid-experimental/experimental-fluidframework": "^0.35.2",
+        "@fluid-experimental/fluid-static": "^0.35.2",
+        "@fluidframework/aqueduct": "^0.35.2",
```

Comment:
Could remove this...

Reply:
yes, we should only need data-objects and fluid-static. even the `experimental-fluidframework` can go